src/KOKKOS/kokkos_type.h

```diff
@@ -25,7 +25,9 @@
 #include <Kokkos_ScatterView.hpp>
 #include <Kokkos_UnorderedMap.hpp>
 
-enum{FULL=1u,HALFTHREAD=2u,HALF=4u};
+constexpr unsigned int FULL = 1u;
```

Comment:
@stanmoore1 this is even more critical than the DPD-REACT change. Should this be `unsigned int` or `unsigned char`? or something else. By using constexpr it becomes a compile time constant, so the compiler may adjust the type if needed where this is used.

Reply:
@akohlmey it should be `unsigned int` since we compare to `int neighflag`.

Reply:
@stanmoore1 Thanks, again. Any particular reason why it is unsigned? It doesn't need the extra bit and CPUs and compilers "like" signed integers more than unsigned and you can even see performance penalties when mixing them.
Thus I've been trained to "use signed unless you required unsigned".

Reply:
Not sure, I believe this was added by Christian Trott before I started working on Kokkos. So if signed int is better that is fine with me.